tool_water_balance/tests/test_waterbalance.py

```diff
@@ -53,17 +125,327 @@ def setUp(self):
+    print("kak")
+
+
+def test_wb_widget_get_io_series_net(wb_widget):
+    io_series_net = wb_widget._get_io_series_net()
+    expected = [
+        {
+            "label_name": "2D flow to 1D (all domains)",
+            "in": ["1d__1d_2d_flow_in", "2d__1d_2d_flow_in"],
+            "out": ["1d__1d_2d_flow_out", "2d__1d_2d_flow_out"],
+            "type": "NETVOL",
+        },
+        {
+            "label_name": "leakage",
+            "in": ["leak"],
+            "out": ["leak"],
+            "type": "2d_groundwater",
+        },
+        {
+            "label_name": "constant infiltration",
+            "in": ["infiltration_rate_simple"],
+            "out": ["infiltration_rate_simple"],
+            "type": "2d",
+        },
+        {"label_name": "2D flow", "in": ["2d_in"], "out": ["2d_out"], "type": "2d"},
+        {"label_name": "1D flow", "in": ["1d_in"], "out": ["1d_out"], "type": "1d"},
+        {
+            "label_name": "groundwater flow",
+            "in": ["2d_groundwater_in"],
+            "out": ["2d_groundwater_out"],
+            "type": "2d_groundwater",
+        },
+        {
+            "label_name": "lateral flow to 2D",
+            "in": ["lat_2d"],
+            "out": ["lat_2d"],
+            "type": "2d",
+        },
+        {
+            "label_name": "lateral flow to 1D",
+            "in": ["lat_1d"],
+            "out": ["lat_1d"],
+            "type": "1d",
+        },
+        {
+            "label_name": "2D boundary flow",
+            "in": ["2d_bound_in"],
+            "out": ["2d_bound_out"],
+            "type": "2d",
+        },
+        {
+            "label_name": "1D boundary flow",
+            "in": ["1d_bound_in"],
+            "out": ["1d_bound_out"],
+            "type": "1d",
+        },
+        {
+            "label_name": "0D rainfall runoff on 1D",
+            "in": ["inflow"],
+            "out": ["inflow"],
+            "type": "1d",
+        },
+        {
+            "label_name": "change in storage",
+            "in": ["d_2d_vol", "d_2d_groundwater_vol", "d_1d_vol"],
+            "out": ["d_2d_vol", "d_2d_groundwater_vol", "d_1d_vol"],
+            "type": "NETVOL",
+        },
+        {"label_name": "pump", "in": ["pump_in"], "out": ["pump_out"], "type": "1d"},
+        {"label_name": "rain on 2D", "in": ["rain"], "out": ["rain"], "type": "2d"},
+        {
+            "label_name": "interception",
+            "in": ["intercepted_volume"],
+            "out": ["intercepted_volume"],
+            "type": "2d",
+        },
+        {
+            "label_name": "surface sources and sinks",
+            "in": ["q_sss"],
+            "out": ["q_sss"],
+            "type": "2d",
+        },
+    ]
+    assert io_series_net == expected
+
+
+def test_barmanger_2d_groundwater(wb_widget):
+    io_series_2d_groundwater = wb_widget._get_io_series_2d_groundwater()
+    bm_2d_groundwater = BarManager(io_series_2d_groundwater)
+    assert isinstance(bm_2d_groundwater, BarManager)
```

Comment:
Als je een BarManager instantieert, is het dan nuttig om te testen dat het er eentje is? Of is het ergens anders voor nodig?